instrumentation/jsf/mojarra-1.2/javaagent/mojarra-1.2-javaagent.gradle

```diff
@@ -76,5 +76,5 @@ dependencies {
 
   mojarra2TestImplementation group: 'org.glassfish', name: 'jakarta.faces', version: '2.3.12'
 
-  latestDepTestImplementation group: 'org.glassfish', name: 'jakarta.faces', version: '2+'
+  latestDepTestImplementation group: 'org.glassfish', name: 'jakarta.faces', version: '2.3.14'
```

Comment:
Should muzzle also be changed?

Reply:
Yeah.. doing it now...

But MavenCentral metadata is broken :( we are doomed :(

Reply:
😱

Reply:
😭